From f04340e1be820bf3d12253d12ad445783c8990dc Mon Sep 17 00:00:00 2001
From: Mort Yao <soi@mort.ninja>
Date: Sat, 9 Dec 2017 18:42:04 +0100
Subject: [PATCH] Revert "window: Raise and lower tile match in tandem"

---
 src/core/window.c | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/src/core/window.c b/src/core/window.c
index c6f8432a3..34e2e02c9 100644
--- a/src/core/window.c
+++ b/src/core/window.c
@@ -4939,13 +4939,7 @@ meta_window_raise (MetaWindow  *window)
    * the child windows appropriately.
    */
   if (window->screen->stack == ancestor->screen->stack)
-    {
-      /* If the window has a tile sibling, raise it before raising the window itself */
-      if (window->tile_match)
-        meta_stack_raise (window->tile_match->screen->stack, window->tile_match);
-
-      meta_stack_raise (window->screen->stack, ancestor);
-    }
+    meta_stack_raise (window->screen->stack, ancestor);
   else
     {
       meta_warning (
@@ -4977,10 +4971,6 @@ meta_window_lower (MetaWindow  *window)
   meta_topic (META_DEBUG_WINDOW_OPS,
               "Lowering window %s\n", window->desc);
 
-  /* If the window has a tile sibling, lower it before loweting the window itself */
-  if (window->tile_match)
-    meta_stack_lower (window->tile_match->screen->stack, window->tile_match);
-
   meta_stack_lower (window->screen->stack, window);
 }
 
-- 
2.15.1

